services/operatorsinfo/operatorsinfo_inmemory.go

```diff
@@ -298,8 +300,8 @@ func (ops *OperatorsInfoServiceInMemory) queryPastRegisteredOperatorEventsAndFil
 	go func() {
 		alreadyRegisteredOperatorAddrs, alreadyRegisteredOperatorPubkeys, pubkeysErr = ops.avsRegistryReader.QueryExistingRegisteredOperatorPubKeys(
 			ctx,
-			startBlock,
-			stopBlock,
+			opts.StartBlock,
+			opts.StopBlock,
```

Comment:
this will panic if `nil` is passed as opts. Can we change opts to be `Opts` instead of `*Opts`?

Reply:
in every function